components/survey_monkey/survey_monkey.app.mjs

```diff
@@ -9,16 +9,52 @@ export default {
     survey: {
       type: "string",
       label: "Survey",
-      description: "Survey where the action will be performed.",
+      description: `Select a **Survey** from the list.
+      \\
+      Alternatively, you can provide a custom *Survey ID*.`,
       async options() {
-        const { data } = await this.getSurveys();
+        const surveys = await this.getSurveys();
 
-        return data.map((survey) => ({
+        return surveys.map((survey) => ({
           label: survey.title,
           value: survey.id,
         }));
       },
     },
+    responseId: {
+      type: "string",
+      label: "Response",
+      description: `Select one of the above survey's **Responses** to get details of.
+      \\
+      Alternatively, you can provide a custom *Response ID*.`,
```

Comment:
```suggestion
      description: `Select one of the above survey's **Responses** to get details of. Alternatively, you can provide a custom *Response ID*.`,
```

Reply:
This was done to provide a line break in the prop description - do you think it'd be best to remove that line break then?

Here's a visual comparison of the suggested change:
![image](https://user-images.githubusercontent.com/48412907/173940679-11059fe6-fd44-451f-b911-ce0c857cf976.png)